server/src/main/java/com/broadcom/lsp/cobol/core/model/variables/RenameItem.java

```diff
@@ -29,16 +30,27 @@
  * operations.
  */
 @Value
+@ToString(callSuper = true)
 @EqualsAndHashCode(callSuper = true)
 public class RenameItem extends StructuredVariable {
   private static final int LEVEL_NUMBER = 66;
 
-  public RenameItem(
-      @NonNull String name,
-      @NonNull String qualifier,
-      @NonNull Locality definition,
-      @NonNull List<Variable> renamedChildren) {
-    super(LEVEL_NUMBER, name, qualifier, definition);
-    renamedChildren.forEach(this::addChild);
+  public RenameItem(@NonNull String name, @NonNull String qualifier, @NonNull Locality definition) {
```

Comment:
Just suggestion: LEVEL_NUMBER (66,77,88) started to be used more visible  with variable-type-check, and used many places e.g.  VariableDefinitionDelegate. Better make them CONSTANT.

Reply:
Yes, makes sense